modules/gitops-release-notes-1-3-0.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3_{context}"]
+= Release notes for {gitops-title} 1.3
+
+{gitops-title} 1.3 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="support-matrix-1-3_{context}"]
+== Support matrix
+
+Some features in this release are currently in link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]. These experimental features are not intended for production use.
+
+In the table below, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+Note the following scope of support on the Red Hat Customer Portal for these features:
+
+.Compatibility and Support matrix
```

Comment:
I suggest we drop the table at module level as was done for the Pipeline release notes PR. The information is repeated, and this would reduce maintenance as well as potential errors.